back/app/controllers/web_api/v1/projects_controller.rb

```diff
@@ -171,6 +171,19 @@ def votes_by_input_xlsx
     end
   end
 
+  def reset_participation_data
+    project = Project.find(params[:id])
+    authorize project
+
+    ParticipantsService.new.reset_participation_data(project)
+
+    render json: WebApi::V1::ProjectSerializer.new(
+      project,
+      params: jsonapi_serializer_params,
+      include: [:admin_publication]
+    ).serializable_hash, status: :ok
```

Comment:
```suggestion
    head :no_content
```
unless the FE is actually using it?

Reply:
We are using this, yes :)